deploy/cloudformation/.gitignore

```diff
@@ -1,3 +1,4 @@
-elastic-agent-ec2-dev.yml
+elastic-agent-ec2-dev-cspm.yml
+elastic-agent-ec2-dev-cnvm.yml
```

Comment:
I think you can use `dev-*` here instead, but don't you think it would be useful to commit these files and share them?

Reply:
I've kept the pre-existing logic of ignoring the `dev` generated `yml`.
I'm not against pushing the `dev` templates, but they are auto-generated anyways.